mstranslator.js

```diff
@@ -119,14 +119,18 @@ MsTranslator.prototype.call = function(path, params, fn) {
     res.on('end', function () {
       //remove invalid BOM
       body = body.substring(1, body.length);
+      var data = null;
```

Comment:
Can we move this into the `try` block? Just in case the JSON returned is invalid.